tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@tsconfig/node16/tsconfig.json",
   "compilerOptions": {
-    "outDir": "lib"
+    "outDir": "lib",
+    "target": "es6"
```

Comment:
Unfortunately I cannot upgrade to newer Node.js yet, so changed the target. Is it fine with you?

Reply:
Cool, It will work.